### PR TITLE
Adding rake task for syncing issues

### PR DESCRIPTION
**Pull request by FionaDL**
This PR creates a rake task that allows the user to create a cron job to sync the github issues. It also adds specs for testing this behavior in the issue model and an extra spec for the pill_request model. 

https://github.com/fastruby/dash/issues/8

### CODE REVIEW

File: lib/tasks/scheduler.rake [hunk 1]
```diff
@@ -4,3 +4,11 @@ task :update_pull_requests => :environment do
   PullRequest.sync_with_github
   puts "done."
 end
+
+desc "This task uses the github API to update issues"
+task :update_issues => :environment do
+  puts "Updating issues..."
+  byebug
```
Comment (bronzdoc) [inline lib/tasks/scheduler.rake:11]: Remove `byebug`

Comment (FionaDL) [reply lib/tasks/scheduler.rake:11]: ah, yeah, that's a good plan :)

File: spec/models/issue_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        issue = FactoryBot.create(:issue)
+
+        Issue.sync_with_github
+        expect(Issue.all).to be_empty
+      end
+
+      it "updates issues that are open", :vcr do
+        user_bot = FactoryBot.create(:user,
+                                    name: "ombu-bot",
+                                    id: 1)
```
Comment (bronzdoc) [inline spec/models/issue_spec.rb:17]: In out next pair session we will dedicate the two hours to forget this kind of indentation 😬 


Screen Shot 2020-05-20 at 8 48 10 AM

Comment (FionaDL) [reply spec/models/issue_spec.rb:17]: Yes sir!

File: app/models/issue.rb [hunk 3]
```diff
@@ -16,6 +16,21 @@ def self.create_or_update_issue(issue)
...
   end
 
+  def self.sync_with_github
+    issues = self.where(state: :open)
+    octokit_service = OctokitService.new(ENV["GITHUB_MACHINE_USER_ACCESS_TOKEN"])
+
+    issues.each do |issue|
+      begin
+        updated_issue = octokit_service.get_issue(issue)
+        puts updated_issue.inspect
```
Comment (bronzdoc) [inline app/models/issue.rb:26]: Is this a debugging `puts` here?

Comment (FionaDL) [reply app/models/issue.rb]: Sorry, yes.

File: spec/models/issue_spec.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        Issue.sync_with_github
+        expect(Issue.all).to be_empty
+      end
+
+      it "updates issues that are open", :vcr do
+        user_bot = FactoryBot.create(
+          :user,
+          name: "ombu-bot",
+          id: 1
+          )
```
Comment (bronzdoc) [inline spec/models/issue_spec.rb:19]: This is the indentation I was referring. it seems the last ")" is not indented correcty
```ruby
it "updates issues that are open", :vcr do
  user_bot = FactoryBot.create(
    :user,
    name: "ombu-bot",
    id: 1
  )
end
```

**Comment by bronzdoc** [comment]
That sounds good @arielj!

**Pull request merged**